docker-compose.yml

```diff
@@ -6,5 +6,7 @@ services:
     build: .
     volumes:
     - ./src:/code
```

Comment:
Is this under volumes or codebase?

Reply:
Under volumes. Indenting it further to make it more clear.